File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        
        int len = nums.Length;
        if(len == 0 || len == 1)
            return len;

        int idx = 0;
        for(int i = 0; i < len - 1; i++) {
            if(nums[i] != nums[i + 1]) {
                nums[idx++] = nums[i];
            }
        }
        nums[idx++] = nums[len - 1];
        return idx;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        index = 0

        for i in range(len(nums) - 1):
            if nums[i] != nums[i + 1]:
                nums[index] = nums[i]
                index += 1

        nums[index] = nums[len(nums) - 1]
        index += 1
        
        return index